FinniversKit/Sources/Components/JobKeyInfo/Subviews/JobKeyInfoView+Separator.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+extension JobKeyInfoView {
+    class Separator: UIView {
+
+        // MARK: - Init
+
+        override init(frame: CGRect) {
+            super.init(frame: frame)
+            setup()
+        }
+
+        required init?(coder: NSCoder) { fatalError() }
+
+        // MARK: - Setup
+
+        private func setup() {
+            backgroundColor = .tableViewSeparator
+            heightAnchor.constraint(equalToConstant: 1 / UIScreen.main.scale).isActive = true
```

Comment:
If possible ad a comment here with explanation why we use exactly 1 / UIScreen.main.scale here. This is not straightforward... Not necessary, ofc.

Reply:
We want the separator to be 1 pixel tall, but since the constant for the constraint refers to points we need to divide the number of pixels we want by the current scale of the screen to account for higher resolution devices (2x, 3x).

We use it multiple places in both the app and FinniversKit/FinnUI so what we probably should do is to pull it out into a property on `CGFloat`.